src/org/omegat/core/team2/impl/GITRemoteRepository2.java

```diff
@@ -317,12 +318,12 @@ public void switchToVersion(String version) throws Exception {
      */
     @Override
     public void addForCommit(String path) throws Exception {
-        Log.logInfoRB("GIT_START", "addForCommit");
+        LOGGER.atInfo().log(Log.getMessage("GIT_START", "addForCommit"));
         try (Git git = new Git(repository)) {
             git.add().addFilepattern(path).call();
-            Log.logInfoRB("GIT_FINISH", "addForCommit");
+            LOGGER.atInfo().log(Log.getMessage("GIT_FINISH", "addForCommit"));
```

Comment:
Doing this Log.getMessage will be called even if log is not done
What about
` LOGGER.atInfo().log(OStrings.getString("GIT_FINISH"), "addForCommit") ` 
Yes it requires change in the syntax in the bundle, but from what I understand, the log(varargs) method is here to prevent doing string format when not strictly necessary

Reply:
I think we should not change syntax because  all L10N project requires aware of it. It is not realistic.
It is because I introduce public helper  `Log.getMessage` method.

Reply:
Do we really need to log GIT_START GIT_FINISH in INFO level?  There are mostly debug message.

Reply:
```suggestion
            LOGGER.atDebug().log("Git 'addForCommit' execution start"));
```

Reply:
When log is INFO level, we can use `Log.logInfoRB`. so Only a debug level should tweaked.

Reply:
> Do we really need to log GIT_START GIT_FINISH in INFO level? There are mostly debug message.

If this operation takes time then it makes sense to use INFO level in particular when this is run under a command-line script. If it is a short operation (and I strongly suspect it is the case) then DEBUG sounds more appropriate.

Reply:
I'd like change these three messages to be debug level.
```
    private static final String GIT_START_MSG = "Git '{}' execution start";
    private static final String GIT_FINISH_MSG = "Git '{}' execution finished successfully";
    private static final String GIT_NO_CHANGES_MSG = "Git '{}' did nothing because there were no changes";
```

Reply:
About the level, it is ok, see previous comments
But is there a reason why to abandon I18N for them?

Reply:
Debug log is not localized as our codeing standard.

https://sourceforge.net/p/omegat/mailman/message/32940574/